ray-operator/test/sampleyaml/rayjob_test.go

```diff
@@ -17,9 +17,6 @@ func TestRayJob(t *testing.T) {
 		{
 			name: "ray-job.custom-head-svc.yaml",
 		},
-		{
-			name: "ray-job.modin.yaml",
```

Comment:
For reference, this is removed here because it was removed from master but not in release-1.3 yet: https://github.com/ray-project/kuberay/pull/3135